mula/pyproject.toml

```diff
@@ -61,3 +58,6 @@ omit = [
     "scheduler/utils/*",
     "scheduler/__main__.py",
 ]
+
+[tool.pytest.ini_options]
+addopts = "--cov scheduler/ --cov-config=pyproject.toml --cov-report xml --cov-branch --cov-report=term-missing:skip-covered"
```

Comment:
Added such that coverage from pytest doesn't include unnecessary folders/files to count towards coverage